### ISSUE TITLE
How to substitute variables from a build.gradle script?

### ISSUE THREAD

**Issue by green-green-avk** [LikeTheSalad/android-string-reference#4]
This project looks pretty good... However how to substitute the `applicationId` variable from a `build.gradle` script?

**build.gradle**
```
...
android {
...
    applicationVariants.all { variant ->
        variant.resValue "string", "versionName", variant.versionName
        variant.resValue "string", "applicationId", variant.applicationId
    }
...
}
...
```

***some*.xml**
```
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <string name="template_desc_fav_token_help"><![CDATA[
...
        <b>Package:</b>
        <clipboard>${applicationId}</clipboard>
...
    ]]></string>
</resources>
```

**Build error message:** `Key applicationId is missing in the map.`

*But:*

**build/generated/res/resValues/debug/values/gradleResValues.xml**
```
<?xml version="1.0" encoding="utf-8"?>
<resources>

    <!-- Automatically generated file. DO NOT MODIFY -->

    <!-- Values from the variant -->
    <string name="applicationId" translatable="false">green_green_avk.anotherterm.debug</string>
    <string name="versionName" translatable="false">MkIIIw-dev</string>

</resources>
```

**Comment by LikeTheSalad** [comment]
Hi @green-green-avk , thanks for your feedback :slightly_smiling_face: 

At the moment this plugin only takes resources from your src folder, however, it's a good point that it should also take into account Gradle's generated ones as well. I'll add support for them in the next release :+1: 

Thanks.

**Comment by green-green-avk** [comment]
Why not to add a variable like `manifestPlaceholders` to add the values from a build script directly?
(<https://developer.android.com/studio/build/manifest-build-variables>)

**Comment by LikeTheSalad** [comment]
That's not a bad idea! however, since this plugin is already built for xml inputs, it'd be more natural to get the generated xml strings. It would also make things smoother when it comes to getting variant-specific generated strings, like the ones in your example, as they can be found on the `resValues` dir ordered by variant name.

Thanks!

**Comment by LikeTheSalad** [comment]
Hi @green-green-avk - I've just released version `1.2.1` which has support for gradle-generated strings :)

Again thanks for your feedback! and please let me know if you happen to see another issue.

Regards,
Cesar.

**Comment by green-green-avk** [comment]
Thank you for the quick solution. It works!
